### PR TITLE
Bring CLI docs up to date

### PR DESCRIPTION
**Pull request by corylanou**

### CODE REVIEW

File: source/docs/v0.9/clients/shell.md [hunk 1]
```diff
@@ -153,6 +184,149 @@ Exit will exit the shell
...
+    ]
+}
+```
+
+### Dumping the database
+
+The CLI allows for you to dump your existing database to the same JSON that the [HTTP API endpoint can write to data](http://influxdb.com/docs/v0.9/concepts/reading_and_writing_data.html).
+
+```
+$ influx -dump=true -database=foo
```
Comment (otoolep) [inline source/docs/v0.9/clients/shell.md]: This seems like too much test. How about just 1 or 2 lines, and saying etc etc? Or something?

**Comment by otoolep** [comment]
If you say it's correct, +1. I think the examples might be too verbose though.

File: source/docs/v0.9/clients/shell.md (hunk 1 above)
Comment (corylanou) [inline source/docs/v0.9/clients/shell.md]: Agreed.  Shortened.

File: source/docs/v0.9/clients/shell.md [hunk 2]
```diff
@@ -10,10 +10,13 @@ There are several arguments you can pass into the shell when starting.  You can
...
 Usage of default:
   -database="": database to connect to the server.
+  -dump=false: dump the contents of the given database to stdout
+  -execute="": Execute command and quit.
+  -format="column": format specifies the format of the server responses:  json, csv, or column
   -host="localhost": influxdb host to connect to
-  -output="column": format specifies the format of the server responses:  json, csv, or column
   -password="": password to connect to the server.  Leaving blank will prompt for password (--password="")
   -port=8086: influxdb port to connect to
+  -pretty=false: turns on pretty print for the json format
```
Comment (beckettsean) [inline source/docs/v0.9/clients/shell.md]: To me, this line implies that `-pretty=false` turns on pretty-printing. I think something like `-pretty=[ true | false ]: control pretty printing of the JSON` or `-pretty=true: turn on pretty print for JSON output, set to false to disable` or even just `-pretty=true: turns on pretty print for the json format`.

File: source/docs/v0.9/clients/shell.md [hunk 3]
```diff
@@ -10,10 +10,13 @@ There are several arguments you can pass into the shell when starting.  You can
 $ influx --help
 Usage of default:
   -database="": database to connect to the server.
+  -dump=false: dump the contents of the given database to stdout
```
Comment (beckettsean) [inline source/docs/v0.9/clients/shell.md]: I think this should be `-dump=true: dump the contents of the given database to stdout`.

File: source/docs/v0.9/clients/shell.md [hunk 4]
```diff
@@ -99,9 +102,35 @@ For a complete reference to the query language, please read the [online document
 
 ```sh
 > show databases
-name    tags    name
-----    ----    ----
-                foo
+name: databases
+---------------
+name
```
Comment (beckettsean) [inline source/docs/v0.9/clients/shell.md]: This syntax confuses me. It happens for `show measurements` and `show databases`. The word "name" appears too many times and the second appearance looks like it's a database name, not the header row to a list of database names. I think it's much clearer to do either 
```
name: databases
---------------
<dbname>
```
or
```
name: databases
---------------
<dbname>
```
Best would be 
```
```
databases
name
---------------
<dbname>
```
but I assume that's hard because you're trying to use a universal format for results.

File: source/docs/v0.9/clients/shell.md [hunk 5]
```diff
@@ -153,6 +184,80 @@ Exit will exit the shell
...
+                }
+            ]
+        }
+    ]
+}
+```
+
+### Dumping the database
+
+The CLI allows for you to dump your existing database to the same JSON that the [HTTP API endpoint can write to data](http://influxdb.com/docs/v0.9/concepts/reading_and_writing_data.html).
```
Comment (beckettsean) [inline source/docs/v0.9/clients/shell.md]: Minor clarity edit, but I would word this as:

"The CLI allows you to dump your existing database to a JSON file that is valid for writing into InfluxDB via the [HTTP API endpoint](http://influxdb.com/docs/v0.9/concepts/reading_and_writing_data.html).

File: source/docs/v0.9/clients/shell.md (hunk 4 above)
Comment (corylanou) [inline source/docs/v0.9/clients/shell.md]: I agree, but this is not related to the CLI, it's related to the output we get from Influx.  I believe we need to go through several of our "meta" responses and correctly name the json.  For instance, for that call, here is the actual json that comes back:

```json
{
    "results": [
        {
            "series": [
                {
                    "name": "databases",
                    "columns": [
                        "name"
                    ],
                    "values": [
                        [
                            "foo"
                        ]
                    ]
                }
            ]
        }
    ]
}
```

Here is what comes back on `show tag keys`

```json
{
    "results": [
        {
            "series": [
                {
                    "name": "cpu",
                    "columns": [
                        "tagKey"
                    ]
                },
                {
                    "name": "names",
                    "columns": [
                        "tagKey"
                    ]
                },
                {
                    "name": "sensor",
                    "columns": [
                        "tagKey"
                    ]
                }
            ]
        }
    ]
}
```

File: source/docs/v0.9/clients/shell.md (hunk 2 above)
Comment (corylanou) [inline source/docs/v0.9/clients/shell.md]: This was using the default `usage` that Go will emit.  I updated with custom usage now.

File: source/docs/v0.9/clients/shell.md (hunk 3 above)
Comment (corylanou) [inline source/docs/v0.9/clients/shell.md]: This was using the default `usage` that Go will emit.  I updated with custom usage now.

File: source/docs/v0.9/clients/shell.md (hunk 4 above)
Comment (beckettsean) [inline source/docs/v0.9/clients/shell.md]: I figured it was an external constraint but wanted to raise it here. I'm fine with it being what it is for now, but yes, we should make the responses easier to visually parse. Will save plugin authors a lot of headache, too.

**Comment by beckettsean** [comment]
+1
